src/prefect/tasks/soda/soda.py

```diff
@@ -0,0 +1,79 @@
+from sodasql.scan.scan_builder import ScanBuilder
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+from typing import Dict, Union
+
+
+class SodaSQLScan(Task):
+    """
+    SodaSQLScan
+    """
+
+    def __init__(
+        self,
+        scan_def: Union[Dict, str] = None,
+        warehouse_def: Union[Dict, str] = None,
+        **kwargs
+    ):
+        """
+
+        Args:
+            scan_def:
+            warehouse_def:
+            **kwargs:
+        """
```

Comment:
Docstrings for `__init__` should go on the class itself (above) rather than the `__init__` method.